include/Dialects/LinalgTransform/TrackingCSE.h

```diff
@@ -9,13 +9,11 @@
 #ifndef MLIR_DIALECT_LINALG_TRANSFORMS_TRACKINGCSE_H
 #define MLIR_DIALECT_LINALG_TRANSFORMS_TRACKINGCSE_H
 
+#include "mlir/Dialect/Linalg/IR/LinalgInterfaces.h"
```

Comment:
I wouldn't want this dependency unless strictly necessary.

Reply:
`SmallVector<LinalgOp>` requires the type to be complete, so unfortunately a forward declaration won't work.